examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,231 @@ Documentation for other releases can be found at
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+##Table of Contents
+
+  - [Cluster Requirements](#cluster-requirements)
+  - [Decide where you will store your data](#decide-where-you-will-store-your-data)
+    - [GCE Persistent Disk](#gce-persistent-disk)
+    - [Host Path](#host-path)
+  - [Create the MySQL Password Secret](#create-the-mysql-password-secret)
+  - [Deploy MySQL](#deploy-mysql)
+  - [Deploy WordPress](#deploy-wordpress)
+  - [Visit your new WordPress blog](#visit-your-new-wordpress-blog)
+  - [Take down and restart your blog](#take-down-and-restart-your-blog)
+  - [Next Steps](#next-steps)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Cluster Requirements
+
+Consult a
+[Getting Started Guide](../../docs/getting-started-guides/)
+to set up a cluster and the kubectl command line client.
+
+Kubernetes runs in a variety of environments and is inherently
+modular. Not all clusters are the same. These are the requirements for
+this example.
+
+* Kubernetes version 1.2 is required due to using newer features, such
+  at PV Claims and Deployments.
+* [Cluster DNS](../../cluster/addons/dns/) will be used for service discovery.
+* An [external load balancer](../../docs/user-guide/services.md#type-loadbalancer)
+  will be used to access WordPress.
+* [Persistent Volume Claims](../../docs/user-guide/persistent-volumes.md)
+  are used. You must create Persistent Volumes in your cluster to be
+  claimed. This example demonstrates how to create two types of
+  volumes, but any volume is sufficient.
+
+## Decide where you will store your data
+
+Both MySQL and WordPress will have a
+[Persistent Volume](../../docs/user-guide/persistent-volumes.md) to
+store their data. We will use a Persistent Volume Claim to claim an
+available persistent volume. This example covers HostPath and
+GCEPersistentDisk volumes. Choose one of the two, or see
+[Types of Persistent Volumes](../../docs/user-guide/persistent-volumes.md#types-of-persistent-volumes)
+for more options.
+
+### GCE Persistent Disk
+
+Please see the
+[GCE getting started guide](../../docs/getting-started-guides/gce.md)
+for full details and other options for starting a cluster.
+
+Create two persistent disks. You will need to create the disks in the
+same [GCE zone](https://cloud.google.com/compute/docs/zones) as the
+Kubernetes cluster. The default setup script will create the cluster
+in the `us-central1-b` zone, as seen in the
+[config-default.sh](../../cluster/gce/config-default.sh) file. Replace
+`<zone>` below with the appropriate zone. The names `wordpress-1` and
+`wordpress-2` must match the `pdName` fields we have specified in
+[gce-volumes.yaml](gce-volumes.yaml).
+
+```shell
+gcloud compute disks create --size=20GB --zone=<zone> wordpress-1
+gcloud compute disks create --size=20GB --zone=<zone> wordpress-2
```

Comment:
The purpose of creating two PVs here is to demonstrate that the PVC binder will choose one at random?